### PR TITLE
added Show view for User

### PR DESCRIPTION
**Pull request by JuanSaieh**
Changes:
- Added Bootstrap gem to project
- Added User index view to show items in a table
- ScreenShot:
Screen Shot 2019-08-01 at 5 01 46 PM

### CODE REVIEW

**Comment by JuanSaieh** [comment]
Screen Shot 2019-08-01 at 5 01 46 PM

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ def create
     end
   end
 
+  def show
+    set_user
+  end
+
   def user_params
     user_params = params.require(:user).permit(:first_name,
```
Comment (Jucerweb) [inline app/controllers/users_controller.rb]: Could you fix this indentation ?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -19,6 +19,10 @@ def create
     end
   end
 
+  def show
+    set_user
```
[the remark this answers is not in the document]
Comment (JuanSaieh) [reply app/controllers/users_controller.rb]: done!

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (JuanSaieh) [reply app/controllers/users_controller.rb]: Of course!

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -28,4 +32,8 @@ def user_params
       :address
       )
   end
+
+  def set_user
+    @user = User.find(params[:id])
+  end
```
Comment (Jucerweb) [inline app/controllers/users_controller.rb]: Add a space

File: config/routes.rb [hunk 4]
```diff
@@ -2,7 +2,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :products, only: [:index, :destroy]
 
-  resources :users, only: [:index, :new, :create]
+  resources :users, only: [:index, :new, :create, :show]
```
Comment (Jucerweb) [inline config/routes.rb]: Change `only` for `except`

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (JuanSaieh) [reply app/controllers/users_controller.rb]: Done!

File: config/routes.rb (hunk 4 above)
Comment (JuanSaieh) [reply config/routes.rb]: Done!

**Pull request merged**